lib/validate.js

```diff
@@ -648,14 +650,14 @@ function checkPathItem(pathItem, path, openapi, options) {
         options.context.pop();
     }
 
-    for (let o in pathItem) {
+    for (const o in pathItem) {
         contextAppend(options, o);
         var op = pathItem[o];
         if (o === '$ref') {
             should(op).be.ok();
             op.should.have.type('string');
             should(op.startsWith('#/')).equal(false,'PathItem $refs must be external ('+op+')');
-            options.linter('reference',op,options);
+            options.linter('reference', pathItem, options);
```

Comment:
The actual bug 😅

Reply:
The actual bug was actually a bug and this is a good fix for it. I.e. the object containing the `$ref` string should be passed to the linter, not the string itself.